src/robot.coffee

```diff
@@ -226,7 +227,7 @@ class Robot
   #
   # middleware - A function that determines whether or not a given matching
   #              Listener should be executed. The function is called with
-  #              (robot, listener, response, next, done). If execution should
+  #              (context, next, done). If execution should
```

Comment:
I was wondering about this change, but i'm pretty sure it was just a stale doc from before we had a context object.